website/docs/r/datadog_monitors.html.markdown

```diff
@@ -55,62 +55,62 @@ The following arguments are supported:
 
 ---
 
 A `datadog_organization` block exports the following:
 
-* `api_key` - (Required) Api key associated to the Datadog organization. Changing this forces a new datadog Monitor to be created.
+* `api_key` - (Required) Api key associated to the Datadog organization. Changing this forces a new Datadog Monitor to be created.
 
-* `application_key` - (Required) Application key associated to the Datadog organization. Changing this forces a new datadog Monitor to be created.
+* `application_key` - (Required) Application key associated to the Datadog organization. Changing this forces a new Datadog Monitor to be created.
 
 * `enterprise_app_id` - (Optional) The ID of the enterprise_app.
 
-* `linking_auth_code` - (Optional) The auth code used to linking to an existing datadog organization.
+* `linking_auth_code` - (Optional) The auth code used to linking to an existing Datadog organization. Changing this forces a new Datadog Monitor to be created.
 
-* `linking_client_id` - (Optional) The ID of the linking_client.
+* `linking_client_id` - (Optional) The ID of the linking_client. Changing this forces a new Datadog Monitor to be created.
 
-* `redirect_uri` - (Optional) The redirect uri for linking.
+* `redirect_uri` - (Optional) The redirect uri for linking. Changing this forces a new Datadog Monitor to be created.
 
 ---
 
 An `identity` block supports the following:
 
-* `type` - (Required) Specifies the identity type of the datadog Monitor. At this time the only allowed value is `SystemAssigned`.
+* `type` - (Required) Specifies the identity type of the Datadog Monitor. At this time the only allowed value is `SystemAssigned`.
 
-> **NOTE:** The assigned `principal_id` and `tenant_id` can be retrieved after the identity `type` has been set to `SystemAssigned` and the datadog Monitor has been created. More details are available below.
+> **NOTE:** The assigned `principal_id` and `tenant_id` can be retrieved after the identity `type` has been set to `SystemAssigned` and the Datadog Monitor has been created. More details are available below.
 ---
 
 An `user` block exports the following:
 
 * `name` - (Required) The name which should be used for this user_info.
 
-* `email` - (Required) Email of the user used by Datadog for contacting them if needed. Changing this forces a new datadog Monitor to be created.
+* `email` - (Required) Email of the user used by Datadog for contacting them if needed. Changing this forces a new Datadog Monitor to be created.
 
 * `phone_number` - (Optional) Phone number of the user used by Datadog for contacting them if needed.
 
 ## Attributes Reference
 
 In addition to the Arguments listed above - the following Attributes are exported:
 
-* `id` - The ID of the datadog Monitor.
+* `id` - The ID of the Datadog Monitor.
 
 * `identity` - A `identity` block as defined below.
 
-* `resource_category` - The catogory of resource logs flowing.
+* `resource_category` - The category of resource logs flowing.
```

Comment:
We should still fix this up. I'm not sure what this is referring to

Reply:
Updated, removed the property as customer won't use it.